src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Templating\Tests\Storage\FileStorageTest;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class DownloadBuildCommand extends Command
+{
+    protected static $defaultName = 'sulu:admin:download-build';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @var string
+     */
+    private $projectDir;
+
+    /**
+     * @var string
+     */
+    private $suluVersion;
+
+    /**
+     * @var string
+     */
+    private $remoteRepository;
+
+    /**
+     * @var string
+     */
+    private $remoteArchive;
+
+    const ASSETS_DIR = DIRECTORY_SEPARATOR . 'assets' . DIRECTORY_SEPARATOR . 'admin' . DIRECTORY_SEPARATOR;
+
+    const BUILD_DIR = DIRECTORY_SEPARATOR . 'public' . DIRECTORY_SEPARATOR . 'build' . DIRECTORY_SEPARATOR . 'admin';
+
+    const REPOSITORY_NAME = 'skeleton';
+
+    public function __construct(HttpClientInterface $httpClient, string $projectDir, string $suluVersion)
+    {
+        parent::__construct();
+
+        $this->httpClient = $httpClient;
+        $this->projectDir = $projectDir;
+        $this->suluVersion = $suluVersion;
+        $this->remoteRepository = 'https://raw.githubusercontent.com/sulu/skeleton/' . $suluVersion;
+        $this->remoteArchive = 'https://codeload.github.com/sulu/skeleton/zip/' . $suluVersion;
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Downloads the current admin application build from the sulu/skeleton repository.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln('<info>Checking for changed files...</info>');
```

Comment:
You can use SymfonyStyle class to simplify outputs: https://symfony.com/doc/current/console/style.html#basic-usage.